### ISSUE TITLE
[Feature] Removing Shulkerboxes from the ban list

### ISSUE THREAD

**Issue by Bebrich** [MrCrayfish/Backpacked#219]
I wish I could remove shulkerboxes from the banned items list, but they seem to be hard-coded into the mod, and I simply cant. My list is empty, yet I still cant put them inside of my backpack.
![image](https://github.com/user-attachments/assets/ae3f5605-505f-420d-ab89-65b4fb105b6f)

**Comment by Dumpling00** [comment]
Think this is to avoid people putting backpacks inside shulkers and then inside other backpacks and creating essentially infinite storage.

**Comment by LadyStrella** [comment]
> Think this is to avoid people putting backpacks inside shulkers and then inside other backpacks and creating essentially infinite storage.

there's already settings for not being able to unequip a backpack with items, if that is enabled then it wouldnt be a problem

**Comment by ks-by** [comment]
Yeah this is weird. I want the option to remove it either. With a lot of mods and resources it have become really unhandy. Please add it to the config.
